boards/shields/semtech_sx1272mb2das/semtech_sx1272mb2das.overlay

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020 Lemonbeat GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	status = "okay";
+
+	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>;	/* D10 */
+
+	lora : sx1272@0 {
+		status = "ok";
```

Comment:
It is not needed explicitly but if you want to do, please use `status = "okay"`

Reply:
changed to "okay"